.circleci/config.yml

```diff
@@ -408,15 +408,18 @@ jobs:
           paths:
             - ./build
 
-  tests.unit:
+  tests.unit.algorithms:
     <<: *defaults
     steps:
       - attach_workspace:
           at: /hpx
       - run:
-          name: Building Unit Tests
+          name: Building Unit Tests (Algorithms)
           command: |
-              ninja -j2 -k 0 tests.unit
+              ninja -j2 -k 0 \
+                tests.unit.modules.algorithms.algorithms \
+                tests.unit.modules.algorithms.block \
+                tests.unit.modules.algorithms.datapar_algorithms
```

Comment:
One of the reasons to wait for the split to fix these was that the set of tests will be smaller. None of these tests will run for this repository after the split anymore.

Reply:
Sure. However, I already had to fix several things we've overlooked since CircleCI limited the runtime.